docs/Linuxbrew.md

```diff
@@ -0,0 +1,99 @@
+
+[Features](#features), [dependencies](#dependencies) and [installation instructions](#install) are described below. Terminology (e.g. the difference between a Cellar, Tap, Cask and so forth) is [explained in the documentation](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology).
+
+# Install
+
+The installation script installs Linuxbrew to `/home/linuxbrew/.linuxbrew` if possible and in your home directory at `~/.linuxbrew` otherwise. Using `/home/linuxbrew/.linuxbrew` allows the use of more binary packages, called bottles, than installing in your personal home directory.
+
+Paste at a Terminal prompt:
+
+```sh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"
+```
+
+Follow the *Next steps* instructions to add Linuxbrew to your `PATH` and to your bash shell profile script, either `~/.profile` on Debian/Ubuntu or `~/.bash_profile` on CentOS/Fedora/RedHat.
+
+```sh
+test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
+test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
+test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile
+echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.profile
+```
+
+You're done! Try installing a package:
+
+```sh
+brew install hello
+```
+
+If you're using an older distribution of Linux, installing your first package will also install a recent version of `glibc` and `gcc`. Use `brew doctor` to troubleshoot common issues.
+
+# Features
+
++ Can install software to your home directory and so does not require sudo
++ Install software not packaged by your host distribution
++ Install up-to-date versions of software when your host distribution is old
++ Use the same package manager to manage your macOS, Linux, and Windows systems
+
+# Dependencies
+
++ **GCC** 4.4 or newer
++ **Linux** 2.6.32 or newer
++ **Glibc** 2.12 or newer
++ **64-bit x86** CPU
+
+Paste at a Terminal prompt:
+
+## Debian or Ubuntu
+
+```sh
+sudo apt-get install build-essential curl file git
+```
+
+## Fedora, CentOS, or Red Hat
+
+```sh
+sudo yum groupinstall 'Development Tools' && sudo yum install curl file git
+```
+
+## Raspberry Pi
+
+Linuxbrew can run on Raspberry Pi (32-bit ARM), but no binary packages (bottles) are available. Support for Raspberry Pi is on a best-effort basis. Pull requests are welcome to improve the experience on Raspberry Pi.
+
+## 32-bit x86
+
+Linuxbrew does not currently support 32-bit x86 platforms. It would be possible for Linuxbrew to work on 32-bit x86 platforms with some effort. Pull requests would be welcome if someone were to volunteer to maintain the 32-bit x86 support.
```

Comment:
Personally I'm not sure the "pull requests would be welcome" part; it'd better be to suggest starting a dedicated fork like Tigerbrew and it could be merged back if it's used.

Reply:
Went with…

Linuxbrew does not currently support 32-bit x86 platforms. It would be possible for Linuxbrew to work on 32-bit x86 platforms with some effort. An interested and dedicated person could maintain a fork of Homebrew to develop support for 32-bit x86.